network/gossip.go

```diff
@@ -34,6 +34,15 @@ func (t *Topic) createObj() proto.Message {
 	return message
 }
 
+func (t *Topic) Close() {
```

Comment:
Right now nothing in Edge calls this and in Avail SL it's called only once. It can be safeguarded by wrapping internal implementation inside a `sync.Once`, if that seems necessary. WDYT?

Reply:
If possible, add sync.Once or an atomic with state of the node. any would work

Reply:
I added `atomic.Bool` both to `network.Topic` and `syncer.syncPeerClient`.